app/src/main/java/com/github/sdp/ratemyepfl/database/CourseDatabase.kt

```diff
@@ -0,0 +1,8 @@
+package com.github.sdp.ratemyepfl.database
+
+import com.github.sdp.ratemyepfl.model.items.Course
+import com.github.sdp.ratemyepfl.model.review.CourseReview
+
+interface CourseDatabase {
```

Comment:
We will probably need to refactor, or delete, this interface when merging with the repositories, right ?

Reply:
Yes, of course, it will be replaced by the repositories.

Reply:
Ok perfect, thanks.

Reply:
Also, note that I did not write tests for view models since they will completely change.